src/Raven.Client/Documents/Replication/ReplicationNode.cs

```diff
@@ -49,6 +49,14 @@ public string Url
         /// </summary>
         public bool Disabled { get; set; }
 
+        public enum ReplicationType
+        {
+            External,
+            PullAsSink,
+            Internal,
+            Migration
+        }
+
```

Comment:
would it be better to add in the `ReplicationType` as an abstract here? and simply add it to the `ToJson`?

Reply:
Then you could use `nameof` in `ReplicationHelper` instead of hard coded string:

https://github.com/ravendb/ravendb/pull/18277/files#diff-f66551e45b557dcba2843a6d79fa52911a6178806b7ec4794a146c43ac6cccc7R13

Reply:
done